src/onboarding/onboarding-restore-backup.ts

```diff
@@ -16,46 +17,82 @@ import { navigate } from "../common/navigate";
 
 @customElement("onboarding-restore-backup")
 class OnboardingRestoreBackup extends LitElement {
-  @property({ attribute: false }) public hass!: HomeAssistant;
+  @property({ attribute: false }) public hass?: HomeAssistant;
 
   @property({ attribute: false }) public localize!: LocalizeFunc;
 
   @property() public language!: string;
 
-  @state() public _restoring = false;
+  @state() private _restoring = false;
+
+  @state() private _error?: string;
```

Comment:
Where is this error set?

Reply:
Oh thanks, this was from my jobs implementation